dts/bindings/watchdog/silabs,gecko-wdog.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2019 Interay Solutions B.V.
+# Copyright (c) Oane Kingma
+# SPDX-License-Identifier: Apache-2.0
+
+title: Silicon Labs Gecko Family Watchdog driver
```

Comment:
Please remove the title and update the description per #20793

Reply:
Fixed